sycl/source/detail/graph/graph_impl.cpp

```diff
@@ -742,8 +742,9 @@ exec_graph_impl::enqueueNodeDirect(const sycl::context &Ctx,
   const bool xptiEnabled = xptiTraceEnabled();
   xpti_td *CmdTraceEvent = nullptr;
   uint64_t InstanceID = 0;
-  auto StreamID = detail::getActiveXPTIStreamID();
+  uint8_t StreamID;
```

Comment:
```suggestion
  uint8_t StreamID = 0;
```
Can we please initialize `StreamID`? In the past, I've seen Coverity conservatively complaining about potential use of uninitialized value.